.github/workflows/pythonpackage.yml

```diff
@@ -21,6 +21,14 @@ jobs:
       uses: actions/setup-java@v1
       with:
               java-version: '11'
+    - name: pull .txt files
+      run: |
+        cd capreolus/data
+        rm -rf dummy
+        rm ./*
+        wget https://www.dropbox.com/s/h6iovqwsyyv7660/data.tgz?dl=1
```

Comment:
No longer uses git lfs. The caveat is that we should remember to update the link if folds/qrels ever change or if we add a new dataset. This is not really that much of a big deal since the unit tests only make use of the dummy dataset and the dummy dataset would never really change.